src/app/project/project-documents/project-documents.component.ts

```diff
@@ -234,15 +274,19 @@ export class ProjectDocumentsComponent implements OnInit, OnDestroy {
         if (selectedDocs.length === 1) {
           this.storageService.state.labels = selectedDocs[0].labels;
         }
-        this.router.navigate(['p', this.currentProject._id, 'project-files', 'edit']);
+        this.router.navigate(['p', this.currentProject._id, 'project-files', ('external' === selectedDocs[0].internalExt ? 'edit-link' : 'edit')]);
         break;
       case 'delete':
         this.onDeleteDocument();
         break;
       case 'download':
         this.documentTableData.data.map((item) => {
           if (item.checkbox === true) {
-            promises.push(this.api.downloadDocument(this.documents.filter(d => d._id === item._id)[0]));
+						if ('external' === item.internalExt) {
+							window.open(item.documentFileName, "_blank");
```

Comment:
Links should open only in the same tab unless a form is being filled out or other exceptions: https://www.w3.org/WAI/WCAG21/Techniques/general/G200